src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/BQSRPipelineSpark.java

```diff
@@ -92,6 +97,9 @@ protected void runTool(final JavaSparkContext ctx) {
                     "as getParallelVariants(List) is broken");
         }
         final JavaRDD<GATKRead> initialReads = getReads();
```

Comment:
Have you run BQSR on a large input with this change? I expect it might have some memory issues with the new default of 128 MB partitions on the input...